apps/api/src/app/inbox/inbox.controller.ts

```diff
@@ -195,4 +199,71 @@ export class InboxController {
       })
     );
   }
+
+  @UseGuards(AuthGuard('subscriberJwt'))
+  @Post('/notifications/all-read')
```

Comment:
```suggestion
  @Post('/notifications/mark-all-as-read')
```

If not mistaken, that's where we landed. the DX guide should be updated accordingly.

Reply:
It's not a problem, I'll update it :)